Meetings/2021-11-01-meeting-minutes.md

```diff
@@ -0,0 +1,91 @@
+# Maintainers' Meeting 2021-11-10
+
+## Meeting Attendees
+
+- Rob Prouse
+- Mattias Karlsson
+- Andrew White
+- Calvin A. Allan
+- Dan Siegel
+- Dylan Perks
+- Gary Ewan Park
+- Glenn Watson
+- James Newton-King
+- James Turner
+- Matthias Koch
+- Nils Andresen
+- Pascal Berger
+- Simon Cropp
+- Tanner Gooding
+- Tony Qu
+- Rob Mensching
+
```

Comment:
```suggestion
- Daniel Valadas @valadas 
```

I think it could be nice to link to each member github profile as a basic contact point at least.

Also, I think the filename is wrong (it says the 1st but the text says the 10th for the date)

Reply:
Agree maybe with delimiter

```diff
+ - Daniel Valadas - @valadas 
```